src/credentials.js

```diff
@@ -73,7 +73,7 @@ class Credentials extends EventEmitter {
       }
     });
   }
-  login(captcha = {}) {
+  login(captcha = {}, options = {}) {
```

Comment:
I personally prefer `options, captcha` order, but that'd break backwards compatibility. Since changing error message format breaks backward compatibility, changing it won't matter that much though. What's your opinion about this?